src/emulate.c

```diff
@@ -1356,6 +1410,77 @@ static void block_translate(riscv_t *rv, block_t *block)
     block->ir[block->n_insn - 1].tailcall = true;
 }
 
+#define PACK_MEM_OPS(RW)                                                   \
```

Comment:
The verb "pack" was not precise. Instead, use "combine," which conveys the notion of merging multiple instructions into a single one.